src/components/frame/index.tsx

```diff
@@ -53,7 +53,8 @@ export class SmoothlyFrame {
 				this.contentWindow
 			)
 	}
+
 	render() {
-		return <iframe src={this.url + "#" + window.location.origin} height="100%" width="100%"></iframe>
+		return [] //<iframe src={this.url + "#" + window.location.origin} height="100%" width="100%"></iframe>
```

Comment:
if we moving away from having the iframe component working the current fashion then the comment should be removed as well. Otherwise reinstate this component to the way it used to be.